packages/varlet-ui/src/button/Button.vue

```diff
@@ -71,19 +72,19 @@ export default defineComponent({
     }
 
     const handleClick = (e: Event) => {
-      const { loading, disabled, onClick } = props
+      const { onClick } = props
 
-      if (!onClick || loading || disabled || pending.value) {
+      if (!onClick || notAvailable.value) {
         return
       }
 
       attemptAutoLoading(onClick(e))
     }
 
     const handleTouchstart = (e: Event) => {
-      const { loading, disabled, onTouchstart } = props
+      const { onTouchstart } = props
 
-      if (!onTouchstart || loading || disabled || pending.value) {
```

Comment:
I think it's no necessary.Usually we set loading just be a temporary status. Just like fetch something.Prevent user post again.From the case we can know prop 
loading is mutable.Um so that extra 
the loading and disable as a cache value will not have good effect.Because it's in memory

Reply:
My pr does cause unnecessary loading. Thank you very much for your answer.